tools/mo/openvino/tools/mo/front/tf/RFFTRealImagToRFFTSplit.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from openvino.tools.mo.front.common.partial_infer.utils import int64_array
+from openvino.tools.mo.front.common.replacement import FrontReplacementSubgraph
+from openvino.tools.mo.front.subgraph_matcher import SubgraphMatch
+from openvino.tools.mo.front.tf.graph_utils import create_op_with_const_inputs
+from openvino.tools.mo.graph.graph import Graph, rename_nodes
+from openvino.tools.mo.ops.split import Split
+from openvino.tools.mo.ops.squeeze import Squeeze
+
+
+class RFFTRealImagToRDFTSplit(FrontReplacementSubgraph):
+    """
+    This transformation converts the operation TFRFFT into OpenVINO RDFT.
+    """
+    enabled = True
+
+    def run_before(self):
+        from openvino.tools.mo.front.tf.TFFFTToDFT import TFFFTToDFT
+        return [TFFFTToDFT]
```

Comment:
Check how many times import is used?

Reply:
checked that it is called only once (at least during conversion the model enabled by this PR)